cmd/podman/varlink.go

```diff
@@ -76,6 +76,7 @@ func varlinkCmd(c *cli.Context) error {
 		switch err.(type) {
 		case varlink.ServiceTimeoutError:
 			logrus.Infof("varlink service expired (use --timeout to increase session time beyond %d ms)", c.Int64("timeout"))
+			logrus.Infof("if you want the varlink interface to be up forever, please set timeout to 0")
```

Comment:
Is this going to fill the logs with information?
Man page has "A value of 0 means no timeout and the session will not expire." 
Which I think would be a better comment.

Reply:
this info message is exactly how you wanted it @rhatdan , we went around in circles about this the first time.  I personally think the message is fine as is .  If we felt it needed adjusting, i think adding "A value of 0 means ..." would be the right approach.

Reply:
How about:

logrus.Infof("varlink service expired (use --timeout to increase session time beyond %d ms, 0 means never timeout)", c.Int64("timeout")) 

@baude I am just thinking we are putting this message in the logs every time we get socket activated.  If we have something like cockpit-podman running this could become noisy and add little value.

Could we see if an actual command happened.  And not issue the message, or issue it at Debug level, then only issue it at info level if there was actually nothing that happened.

Reply:
my point here was that I read that message and I did not understand what it meant so I proposed this PR with a more beginner-friendly message

Reply:
@TomasTomecek Simplify it down to one line and then we can merge and we can argue separately about how to handle flooding the log with messages.